apps/judicial-system/backend/src/app/modules/notification/notification.controller.ts

```diff
@@ -81,6 +81,7 @@ export class NotificationController {
         NotificationType.HEADS_UP,
         NotificationType.READY_FOR_COURT,
         NotificationType.COURT_DATE,
+        NotificationType.DEFENDER_ASSIGNED,
```

Comment:
Please consider removing the call to sendCaseNotification below, as all notifications should now be queued.

Reply:
I cleaned up the user types. After doing so I feel like we should change the name of the notification controller method to "sendCaseNotificationToQueue" and adjust the return type, since that's a more accurate representation of what the method is actually doing now. I think that should be a different PR though because it requires quite a few more changes to more files.